### PR TITLE
Split Organization view

### PR DESCRIPTION
**Pull request by Cheshiriks**
Split Organization view

Снимок экрана 2023-12-01 в 16 34 55
Снимок экрана 2023-12-01 в 17 12 18

### CODE REVIEW

File: save-cosv-frontend/src/main/kotlin/com/saveourtool/save/cosv/frontend/components/views/organization/OrganizationView.kt [hunk 1]
```diff
@@ -0,0 +1,471 @@
+/**
```
Comment (akuleshov7) [inline save-cosv-frontend/src/main/kotlin/com/saveourtool/save/cosv/frontend/components/views/organization/OrganizationView.kt:1]: The logic of split was different. We need to make a common community framework constructor:
- Try to create organization View in common 
- We will be able to provide different enums as confiuration on the implementation side (where you will be able to select whcih tabs are needed)

File: save-cosv-frontend/src/main/kotlin/com/saveourtool/save/cosv/frontend/components/views/organization/OrganizationView.kt [hunk 2]
```diff
@@ -0,0 +1,471 @@
...
+    /**
+     * Organization avatar
+     */
+    var avatar: String
+}
+
+/**
+ * A Component for owner view
+ */
+class OrganizationView : AbstractView<OrganizationProps, OrganizationViewState>(Style.SAVE_LIGHT) {
```
[the remark this answers is not in the document]
Comment (Cheshiriks) [reply save-cosv-frontend/src/main/kotlin/com/saveourtool/save/cosv/frontend/components/views/organization/OrganizationView.kt]: Changed it

File: save-cosv-frontend/src/main/kotlin/com/saveourtool/save/cosv/frontend/components/views/organization/OrganizationView.kt (hunk 1 above)
Comment (Cheshiriks) [reply save-cosv-frontend/src/main/kotlin/com/saveourtool/save/cosv/frontend/components/views/organization/OrganizationView.kt]: Fixed

File: save-frontend-common/src/main/kotlin/com/saveourtool/save/frontend/common/components/views/organization/RenderInfoTab.kt [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+            }
+        }
+    }
+
+    // ================= Rows for TOP projects ================
+    val topProjects = props.projects.sortedByDescending { it.contestRating }.take(TOP_PROJECTS_NUMBER)
+
+    if (topProjects.isNotEmpty()) {
+        // ================= Title for TOP projects ===============
+        div {
```
Comment (akuleshov7) [inline save-frontend-common/src/main/kotlin/com/saveourtool/save/frontend/common/components/views/organization/RenderInfoTab.kt:63]: you can move it to tools

File: save-frontend-common/src/main/kotlin/com/saveourtool/save/frontend/common/components/views/organization/OrganizationSettingsMenu.kt [hunk 4]
```diff
@@ -141,48 +141,52 @@ private fun organizationSettingsMenu() = FC<OrganizationSettingsMenuProps> { pro
...
-                                checked = props.organization.canCreateContests
-                                onChange = {
-                                    props.onCanCreateContestsChange(!props.organization.canCreateContests)
+                            className = ClassName("d-sm-flex justify-content-center form-check pl-3 pr-3 pt-3")
+                            div {
+                                input {
+                                    className = ClassName("form-check-input")
+                                    type = InputType.checkbox
+                                    value = props.organization.canCreateContests.toString()
+                                    id = "canCreateContestsCheckbox"
```
Comment (akuleshov7) [inline save-frontend-common/src/main/kotlin/com/saveourtool/save/frontend/common/components/views/organization/OrganizationSettingsMenu.kt:152]: Move super admin settings from a user settings tab to a separate tab, that will be visible only for super admins

File: save-frontend-common/src/main/kotlin/com/saveourtool/save/frontend/common/components/basic/TestSuitesDisplayer.kt [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+import web.cssom.ClassName
+
+/**
+ * @param testSuites
+ * @param selectedTestSuites
+ * @param displayMode if used not inside TestSuiteSelector, should be null, otherwise should be mode of TestSuiteSelector
+ * @param onTestSuiteClick
+ */
+@Suppress("TOO_LONG_FUNCTION", "LongMethod")
+fun ChildrenBuilder.showAvailableTestSuites(
```
[the remark this answers is not in the document]
Comment (Cheshiriks) [reply save-frontend-common/src/main/kotlin/com/saveourtool/save/frontend/common/components/basic/TestSuitesDisplayer.kt:28]: Yes. It's benchmarks tabs

File: save-frontend-common/src/main/kotlin/com/saveourtool/save/frontend/common/components/views/organization/RenderInfoTab.kt (hunk 3 above)
Comment (Cheshiriks) [reply save-frontend-common/src/main/kotlin/com/saveourtool/save/frontend/common/components/views/organization/RenderInfoTab.kt]: Moved it

**Review by akuleshov7 — APPROVED** [review]

**Pull request merged**